src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -654,35 +697,67 @@ private void spawnCitizen(CitizenData data)
                 {
                     //TODO: add Colony Name prefix?
                     LanguageHandler.sendPlayersLocalizedMessage(
-                            EntityUtils.getPlayersFromUUID(world, permissions.getMessagePlayers()),
+                            ServerUtils.getPlayersFromUUID(world, permissions.getMessagePlayers()),
                             "tile.blockHutTownHall.messageMaxSize");
                 }
             }
 
             entity.setColony(this, citizenData);
 
-            entity.setPosition(spawnPoint.getX(), spawnPoint.getY(), spawnPoint.getZ());
+            entity.setPosition(spawnPoint.getX() + 0.5D, spawnPoint.getY() + 0.1D, spawnPoint.getZ() + 0.5D);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.5D to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.1D to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.5D to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)